cmd/kres/cmd/gen.go

```diff
@@ -82,6 +82,13 @@ func runGen() error {
 		return err
 	}
 
+	switch options.CIProvider {
+	case "drone":
+		outputs = append(outputs, output.Wrap[drone.Compiler](drone.NewOutput()))
```

Comment:
can we have both enabled in the first iteration and simply drop Drone at some point?

Reply:
but then drone and gh would push the same image

Reply:
we can disable Drone in the Drone UI